ShareX.ScreenCaptureLib/Forms/RegionCaptureForm.cs

```diff
@@ -539,15 +539,29 @@ private void RegionCaptureForm_FormClosing(object sender, FormClosingEventArgs e
             }
         }
 
+        /* Return true if user confirms editor closure */
         internal bool ShowExitConfirmation()
         {
             bool result = true;
 
             if (IsModified)
             {
                 Pause();
-                result = MessageBox.Show(this, Resources.RegionCaptureForm_ShowExitConfirmation_Text, Resources.RegionCaptureForm_ShowExitConfirmation_ShareXImageEditor,
-                    MessageBoxButtons.YesNo, MessageBoxIcon.Question) == DialogResult.Yes;
+                DialogResult result1 = MessageBox.Show(this, Resources.RegionCaptureForm_ShowExitConfirmation_Text, Resources.RegionCaptureForm_ShowExitConfirmation_SaveChanges,
```

Comment:
You are right, but there is no point of changing that until someone from maintainer will step in.

Reply:
Done. But need to rebase, because most wanted "zoom" feature was added recently: https://github.com/ShareX/ShareX/commit/cd499d73d5d215a317c849ceade221745ed6b302 :)